src/Components/reusables/DiveLogImageCarousel.tsx

```diff
@@ -210,7 +159,9 @@ const DiveLogImageCarousel: FunctionComponent<
         </View>
         <View style={styles.imageCountContainer}>
           <Icon name="image-outline" size={18} color="#FFF" />
-          <Text style={styles.imageCountText}>0/0</Text>
+          <Text style={styles.imageCountText}>{`${focusedImageIndex + 1}/${
+            images.length
```

Comment:
When we have to use default images, I wanted to make it clear. to the user that there are no images attached and what we're displaying are default ones. This is why I used. 0/0

Reply:
i'm okay with 1/1 to keep the code simpler